tests/network/network_service/test_service_config_manifest.py

```
@@ -0,0 +1,43 @@
+import pytest
+
+from tests.network.ip_family_services.libservice import SERVICE_IP_FAMILY_POLICY_SINGLE_STACK
+
+
+@pytest.mark.gating
+class TestServiceConfigurationViaManifest:
+    @pytest.mark.single_nic
+    @pytest.mark.parametrize(
+        "single_stack_service_ip_family, single_stack_service",
```

Ruff (0.14.13)

Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)